packages/ui/src/router/index.js

```diff
@@ -44,19 +36,34 @@ const routes = [
     path: '/:id',
     component: MainContent,
   },
+  {
+    name: 'bonus',
+    path: '/bonus/:id',
+    component: MainContent,
+    beforeEnter: async (to, from, next) => {
+      const localStore = useLocalStore()
+      const player = useStore()
+      const loginInfo = localStore.getToken()
+      if (loginInfo.key && router.currentRoute.value.params.id) {
```

Comment:
@gabaldon which one of these is this expecting?
1. `https://wittypixels.art/?id=fabadaacabada` 
2. `https://wittypixels.art/#/?id=fabadaacabada`
3. `https://wittypixels.art/#/fabadaacabada`
4. Other

Thanks!

Reply:
Ok, I'm stupid, it's clearly `https://wittypixels.art/bonus/fabadaacabada`

Reply:
But is the `#` needed before `/bonus`?